bitcoin/src/psbt/map/input.rs

```diff
@@ -223,6 +223,10 @@ impl PsbtSighashType {
     ///
     /// No guarantees are made as to the standardness or validity of the returned value.
     pub fn to_u32(self) -> u32 { self.inner }
+
+    /// Ambiguous `ALL` sighash type, may refer to either [`EcdsaSighashType::All`]
+    /// or [`TapSighashType::All`].
+    pub const ALL: PsbtSighashType = PsbtSighashType { inner: 0x01 };
```

Comment:
Can this go at the top of the impl block please. Perhaps with the following docs
```rust
    /// Ambiguous `ALL` sighash type, may refer to either [`EcdsaSighashType::All`]
    /// or [`TapSighashType::All`].
    ///
    /// This is equivalent to either `EcdsaSighashType::All.into()` or `TapSighashType::All.into()`.
    /// For sighash types other than `ALL` use the ECDSA or Taproot sighash type directly.
    ///
    /// # Example
    ///
    /// ```
    /// let _ = TapSighashType::AllPlusAnyoneCanPay.into();
    /// ```
    pub const ALL: PsbtSighashType = PsbtSighashType { inner: 0x01 };
```

Reply:
Thanks for the suggestion.

Done in 914261c